CSS/WorkPg.css

```diff
@@ -0,0 +1,115 @@
+.worksection {
+  margin: 24px;
+}
+
+.workheading {
+  text-align: center;
+}
+
+.gymimg {
+  margin-top: 80px;
+  margin-right: none;
+  position: static;
+  width: 527px;
+  height: 437px;
+  top: 0;
+  left: 24px;
+  align-items: center;
+  justify-content: center;
+}
+
+.multiPost {
+  display: grid;
+  grid-template-rows: repeat(4, auto);
+}
+
+.multiPostText1 {
+  margin-bottom: 0%;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.multiPostText2 {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.SePrjBTN {
+  width: 154px;
+  margin-right: 100%;
+  position: static;
+}
+
+.tecnoBTN {
+  align-items: center;
+  list-style: none;
+  display: flex;
+  gap: 5px;
+  justify-content: space-evenly;
+}
```

Comment:
- [ ] Kindly delete the `gap: 5px;` and  `justify-content: space-evenly;` to make them close to each other to match the Figma design.

Your design
![image](https://user-images.githubusercontent.com/35535130/166123380-03d093ff-3c0a-4d26-a39b-931354f411d4.png)

Figma design
![image](https://user-images.githubusercontent.com/35535130/166123386-03e5cc3e-81f2-4ae1-9f47-f8b4b0695c30.png)